include/cudaq/Optimizer/CodeGen/Passes.h

```diff
@@ -36,6 +35,8 @@ void registerConvertToQIRPass();
 void addQIRProfilePipeline(mlir::OpPassManager &pm, llvm::StringRef convertTo);
 
 void addLowerToCCPipeline(mlir::OpPassManager &pm);
+void addWiresetToProfileQIRPipeline(mlir::OpPassManager &pm,
+                                    llvm::StringRef profile);
```

Comment:
nit? I think we might use `mlir::StringRef` in the code more often, but I could be totally wrong. It may even be 50/50. Anyway, just a note about conformity.